vivarium_readdy/processes/readdy_process.py

```diff
@@ -199,51 +141,31 @@ def add_repulsions(self, all_particle_types):
         to enforce volume exclusion
         """
         for type1 in all_particle_types:
-            if type1 not in self.parameters["particle_radii"]:
-                raise Exception(
-                    "Please provide a radius for particle type "
-                    f"{type1} in parameters['particle_radii']"
-                )
             for type2 in all_particle_types:
-                if type2 not in self.parameters["particle_radii"]:
-                    raise Exception(
-                        "Please provide a radius for particle type "
-                        f"{type1} in parameters['particle_radii']"
-                    )
                 self.system.potentials.add_harmonic_repulsion(
                     type1,
                     type2,
                     force_constant=self.parameters["force_constant"],
                     interaction_distance=(
-                        self.parameters["particle_radii"][type1]
-                        + self.parameters["particle_radii"][type2]
+                        self.parameters["particle_radii"].get(type1, 1.0)
```

Comment:
maybe this should use a default size set in `self.parameters` instead of a hardcoded `1.0`?